frontend/public/components/RBAC/role.jsx

```diff
@@ -228,9 +228,7 @@ export const BindingsForRolePage = (props) => {
       canCreate={true}
       createButtonText="Create Binding"
       createProps={{
-        to: `/k8s/${
-          ns ? `ns/${ns}` : 'cluster'
-        }/rolebindings/~new?rolekind=${kind}&rolename=${name}`,
+        to: `/k8s/cluster/rolebindings/~new?rolekind=${kind}&rolename=${name}`,
```

Comment:
Won't this simply break the form for a namespaced Role? I would only expect this URL to work for a ClusterRole.

Reply:
No, it seems ok. The other buttons all use this link already.

Reply:
How does the form know which namespace to get the Role from when we're removing it from the URL?

Reply:
Can you point me to the other buttons that use this link for context?

Reply:
Sure, they're here:

Bindings Create button: https://github.com/openshift/console/blob/master/frontend/public/components/RBAC/bindings.jsx#L291
Role kebab: https://github.com/openshift/console/blob/master/frontend/public/components/RBAC/role.jsx#L35

Reply:
@rebeccaalpert You've found another bug. Those links are incorrect as well. I tested to confirm.

You *must* specify a namespace in addition to the role name if it's a `Role` (as opposed to a `ClusterRole`). The way the page should work is that the namespace is preselected and can't change. Right now you can select any namespace on the create role page, which doesn't make sense. (Selecting a namespace is fine for a `ClusterRole`.)

You could fix this by adding a `namespace` query parameter to the URL for `Role` and lock the choice to that namespace.

Reply:
I updated all three links to use this format (as well as a namespace parameter if available). This will show the resource link dropdown when a namespace is defined and none if it's a ClusterRole or if there's a namespace selected on the Role Bindings page. Let me know if you have other thoughts.

Reply:
@rebeccaalpert Sorry for not getting back sooner, I missed this update.

This change is fine, except it doesn't fix the original bug :/ I think we should make the original bug no plan to fix if we want to do this, then open a second bug for the other pages using the wrong URL for namespaced roles.

Reply:
I closed the original and attached the new bug to this PR.